README.md

```diff
@@ -18,14 +18,22 @@ Development with Docker
 =======================
 We provide a Docker image with necessary dependencies required to compile and test MADlib on PostgreSQL 9.6. You can view the dependency Docker file at ./tool/docker/base/Dockerfile_postgres_9_6. The image is hosted on Docker Hub at madlib/postgres_9.6:latest. Later we will provide a similar Docker image for Greenplum Database.
 
-Some useful commands to use the docker file:
+We provide a script to quickly run this docker image at ./tool/docker_start.sh, which will mount your local madlib directory, build MADlib and run install check on this Docker image. At the end, it will `docker exec` as postgres user. Note that you have to run this script from inside your madlib directory, and you can specify your docker CONTAINER_NAME (default is madlib) and IMAGE_TAG (default is latest). Here is an example:
```

Comment:
I think we should also emphasize that this script has to be run just once. Users can then log into the container using `docker exec -it CONTAINER_NAME bash` command for subsequent runs in the development cycle (if it's still alive, else this script has to be run again anyway).
Add a comment saying use `LaTex` as the `IMAGE_TAG` and use `make design_pdf` command to build the design doc.